docs/reference/contributing/connectivity/WifiInterface.md

```diff
@@ -1,2 +1,44 @@
-## WiFiInterface
+<h2 id="wifi-port">Wi-Fi</h2>
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/wifi.png)<span>Wi-Fi driver</span></span>
+
+This document describes how to port and test an IEEE 802.11 Wi-Fi driver to Mbed OS. There are two types of Wi-Fi drivers in Mbed OS, depending on which protocol layer it implements. Wi-Fi drivers are either a special case of Ethernet driver or external IP stacks. Wi-Fi drivers require configuration from an application and, therefore, implement both the low-level EMAC API or network stack API and the high-level controlling interface API called `WiFiInterface`.
+
+### WifiInterface
+
+The `WifiInterface` class is the controlling API from application to driver.
+You can use this API for configuring security settings, network names and keys and also instantiating connection or disconnection phases from the network.
+
```

Comment:
Some pseudo code or examples would be better here..

Reply:
Added.